docs/install/Knative-custom-install.md

```diff
@@ -368,7 +374,7 @@ commands below.
        --filename https://github.com/knative/serving/releases/download/v0.5.2/monitoring.yaml
      ```
 
-   * To install all three Knative components and the set of Eventing sources
+   - To install all three Knative components and the set of Eventing sources
```

Comment:
Perhaps we should also include the flag in the below examples too?

 `--filename https://github.com/knative/serving/releases/download/v0.5.2/serving.yaml --selector networking.knative.dev/certificate-provider!=cert-manager`

Reply:
I think we need the selector as well as the `Note` statement as above.

Reply:
Done

Reply:
Thanks! I just committed an edit to fix all the missing step numbers.